<launch>

 <!-- Create a coordinate frame for the camera so that it can be shown
      in rviz -->

   <node pkg="tf" type="static_transform_publisher"
   name="world_to_cam" args="0 0 0.5 -1.57 0 -1.57 world camera 100"
   />

   <!--Command line arguments and defaults for launcing the
       ar_track_alvar -->

  <arg name="marker_size" default="6" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="cam_image_topic" default="/usb_cam/image_raw" />
  <arg name="cam_info_topic" default="/usb_cam/camera_info" />
  <arg name="output_frame" default="/camera" />

  
  <!--Start the tracker -->

  <node name="ar_track_alvar" pkg="ar_track_alvar"
  type="individualMarkersNoKinect" respawn="false" output="screen"
  args="$(arg marker_size) $(arg max_new_marker_error) $(arg
  max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg
  output_frame)" />


   <!--Start the camera -->

   <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_name" value="logitech" />
    <param name="camera_frame_id" value="/camera" />
    <param name="io_method" value="mmap"/>
    <param name="camera_info_url" type="string" value="file://$(find
    zeta_rescue)/data/logitech_calibration.yaml" />
  </node>


  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find
  zeta_rescue)/rviz/usb_cam_alvar_demo.rviz" />
  
</launch>
